docker-compose.yml

```diff
@@ -22,17 +22,22 @@ services:
     ports:
       - "5306:3306"
     volumes:
+      - ./.data/mysql:/var/lib/mysql
```

Comment:
Will the MySQL db be removed after "docker-compose down"?

Currently, if I keep running "docker-compose stop", without using "docker-compose down", the MySQL database is persistent

Reply:
> Will the MySQL db be removed after "docker-compose down"?
No by mounting the folder outside the docker container, it will persists through a `docker-compose down` (the contents of `/var/lib/mysql` within the container will be in the folder `.data/mysql` within the project folder so it will not be lost when the container is destroyed)

Reply:
This is a good idea, the only thing that ever got me was that you have to remember to wipe out the database you'll have to delete. We should add this to the README.